ci-operator/config/openshift-kni/numaresources-operator/openshift-kni-numaresources-operator-main.yaml

```diff
@@ -25,7 +25,7 @@ operator:
   - pullspec: quay.io/openshift-kni/numaresources-operator:4.10-snapshot
     with: numaresources-operator
 promotion:
-  name: numaresources-operator
+  name: "4.10"
```

Comment:
@fromanirh I think we need to add the bundle image to the image list as well to enable mirroring the job is failing right now..
https://prow.ci.openshift.org/view/gs/origin-ci-test/logs/periodic-image-mirroring-openshift-kni/1463945756372111360